### DISCUSSION TITLE
How can users find out what has changed in 0.4.20 and 0.4.21

### DISCUSSION THREAD

**Discussion by claremacrae** [chhoumann/quickadd#183] [General]
Hi,

I'm just reviewing updates to my plugins, by going to each repo and looking at:

1. The releases pages since my previous version
2. If nothing in the above, a CHANGES/CHANGELOG page
3. If nothing in the above, the README

I do this to partly guard against breakages, but mainly to make sure I learn about new features.

For QuickAdd, 0.4.20 and 0.4.21, I can't see any info about the changes, is there somewhere else I should be looking please?

**Comment by chhoumann** [comment]
Hi @claremacrae!

Both updates were small hotfixes that fixed CSS classes in various modals.

I usually put changes in the README & sometimes the tagged release changelog, but for minor updates like these, I don't do it.
Why? Because they were meant to fix something that I might have overseen in a previous update, so it would be that update again.

I can clarify this in the README :)

**Reply by claremacrae** [reply]
Thanks @chhoumann 

That makes sense.

> I can clarify this in the README :)

If/when you have time, that would help - many thanks.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

This is fixed structurally now, Clare. Releases are cut by semantic-release, so every version - patch hotfixes included - gets generated notes on the Releases page: https://github.com/chhoumann/quickadd/releases. Feature releases additionally announce themselves in-app with an update modal, so users learn about new features without checking the repo.

Thanks for pushing on this back then; it was a fair ask. Closing as resolved.

**Reply by claremacrae** [reply]
Super. Thanks.
